src/app/backend/handler/apihandler.go

```diff
@@ -1108,6 +1113,27 @@ func (apiHandler *APIHandler) handleDeployFromFile(request *restful.Request, res
 	})
 }
 
+func (apiHandler *APIHandler) handleDeployRollBack(request *restful.Request, response *restful.Response) {
+	k8sClient, err := apiHandler.cManager.Client(request)
+	if err != nil {
+		errors.HandleInternalError(response, err)
+		return
+	}
+
+	rollBackSpec := new(deployment.RollBackSpec)
```

Comment:
Change to `rollbackSpec`. If you look at `deployment.RollbackDeployment` method you see the correct capitalization.

Reply:
Great~